packages/edit-site/src/components/dataviews/add-filter.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	privateApis as componentsPrivateApis,
+	Button,
+	BaseControl,
+	Icon,
+} from '@wordpress/components';
+import { chevronRightSmall } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const {
+	DropdownMenuV2,
+	DropdownSubMenuV2,
+	DropdownSubMenuTriggerV2,
+	DropdownMenuItemV2,
+} = unlock( componentsPrivateApis );
+
+export default function AddFilter( { filters, onChangeView } ) {
+	return (
+		<BaseControl>
+			<DropdownMenuV2
```

Comment:
Hey @ciampo 👋  Is there a way to prevent the DropdownMenu from openining when the trigger is disabled?

https://github.com/WordPress/gutenberg/assets/583546/9ad0d282-9d9a-4542-8da4-c4860943dd10

I can render just the disabled Button when necessary instead of the DropdownMenu, but I thought I'd ask in case I'm missing something.

Reply:
I just went for rendering the button, though I'm still curious if this is the recommended pattern.